python-connectors/README.md

```diff
@@ -0,0 +1,60 @@
+# Running python source connectors and SMTs
+
+## Intro
+
+CSID Labs have developed tools and libraries that allow you to write and run python source connectors and SMTs.
+
+In addition to letting you use your favourite language (python), they are also very simple to use. You can write a `Hello World` source connector with just 2 lines of codes.
+
+More details:
+- Source: https://github.com/confluentinc/csid-python-base
+- How it works: https://docs.google.com/presentation/d/1cdfVa2A5FQXwo5Oxi9FUEscQp2z78SAo5xfvjNNZhIg/
+
+## Running a python source connector
+
+This tutorial uses a complete cluster running on `docker`. Just make sure you have Docker Desktop installed on your computer and you're ready to go!
+
+### Start the cluster
+
+1. Open a terminal and go to the tutorial folder.
+2. Start the cluster: `docker-compose up -d`
+3. Wait for a few seconds and run: `docker-compose ps`. Everything container instance should be marked as running.  
+
+### Start a connector and an SMT
+
+From there, you have to create a connector in the Connect cluster.  
+
+There are 3 examples provided with this tutorial in the `json`folder. They all use a python connector and the last one also includes a python SMT.
+
+1. [connector-fields.json](json/connector-fields.json): this example publishes JSON messages containing 6 fields. This corresponds to the `code/speed_traps.py` script.
+2. [connector-full-text.json](json/connector-full-text.json): this example publishes messages with one big string as the value. This is in the `code/full_text.py` script.
+3. [connector-with-smt.json](json/connector-with-smt.json): this example publishes simple messages and the same script also contains an SMT that is modifying the messages. It can be found in the `code/connector_with_smt.py` script.
+
+To create a connector you can use `curl` or you can use the Control Center UI. Since we already have a terminal open, let's use that.
+
+Run `curl localhost:8083/connectors -X POST --json @./json/connector-fields.json` to create a connector with the example 1 above.
```

Comment:
I had to wait a couple of minutes for this to work - not a big deal but maybe put a line there saying you might have to re-try if it doesn't work at first.  I think it could be due to the fact that my mac is `amd64` and not `arm64` - I did get this warning starting `connect The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64/v2) and no specific platform was requested ` it also took a couple of minutes for the `test-topic-XXX` to show up - but everything worked as expected in the end.